.ci/linux-ubuntu16.04-gcc-sgx-apps.jenkinsfile

```diff
@@ -0,0 +1,19 @@
+node('sgx_slave') {
```

Comment:
Instead of hard coding the node label in the JenkinsFile you can pass it as a parameter from the Jenkins job and read it here like this:

agent {
        node {
            label nodelabel
        }
    }